Cargo.toml

```diff
@@ -52,11 +76,12 @@ path = "examples/rust/tcp-ping-pong.rs"
 default = [ "catnap-libos" ]
 catnap-libos = [ ]
 catpowder-libos = [ ]
-catcollar-libos = [ "inetstack/liburing" ]
-catnip-libos = [ "inetstack/libdpdk" ]
-mlx4 = ["inetstack/libdpdk", "inetstack/mlx4"]
-mlx5 = ["inetstack/libdpdk", "inetstack/mlx5"]
-profiler = [ "inetstack/profiler" ]
+catcollar-libos = [ "liburing" ]
+catnip-libos = [ "libdpdk" ]
+libdpdk = [ "dpdk-rs" ]
+mlx4 = [ "dpdk-rs/mlx4" ]
+mlx5 = [ "dpdk-rs/mlx5" ]
+profiler = [  ]
 
 
 [profile.release]
```

Comment:
Inetstack's Cargo.toml (which appears to have been folded into here) also had a "profile.dev" for debug builds.  Do we not want that here?  Or is it covered by something somewhere else?

Reply:
Well noted. I'm working on this.